compose/README.md

```diff
@@ -67,22 +14,39 @@ vitess/examples/compose$ docker-compose up -d
-compose_consul1_1            docker-entrypoint.sh agent ...   Up             8300/tcp, 8301/tcp, 8301/udp, 8302/tcp, 8302/udp, 0.0.0.0:8400->8400/tcp, 0.0.0.0:8500->8500/tcp, 0.0.0.0:8600->8600/tcp, 8600/udp
-compose_consul2_1            docker-entrypoint.sh agent ...   Up             8300/tcp, 8301/tcp, 8301/udp, 8302/tcp, 8302/udp, 8400/tcp, 8500/tcp, 8600/tcp, 8600/udp
-compose_consul3_1            docker-entrypoint.sh agent ...   Up             8300/tcp, 8301/tcp, 8301/udp, 8302/tcp, 8302/udp, 8400/tcp, 8500/tcp, 8600/tcp, 8600/udp
-compose_external_db_host_1   docker-entrypoint.sh mysqld      Up (healthy)   0.0.0.0:32835->3306/tcp, 33060/tcp
-compose_schemaload_1         sh -c /script/schemaload.sh      Exit 0
-compose_vreplication_1       sh -c [ $EXTERNAL_DB -eq 1 ...   Exit 0
-compose_vtctld_1             sh -c  /vt/bin/vtctld -top ...   Up             0.0.0.0:32836->15999/tcp, 0.0.0.0:15000->8080/tcp
-compose_vtgate_1             sh -c /vt/bin/vtgate -topo ...   Up             0.0.0.0:15306->15306/tcp, 0.0.0.0:32845->15999/tcp, 0.0.0.0:15099->8080/tcp
-compose_vtorc_1              sh -c /script/vtorc-up.sh        Up (healthy)   0.0.0.0:13000->3000/tcp
-compose_vttablet100_1        sh -c [ $EXTERNAL_DB -eq 1 ...   Exit 0
-compose_vttablet101_1        sh -c /script/vttablet-up. ...   Up (healthy)   0.0.0.0:32838->15999/tcp, 0.0.0.0:32840->3306/tcp, 0.0.0.0:15101->8080/tcp
-compose_vttablet102_1        sh -c /script/vttablet-up. ...   Up (healthy)   0.0.0.0:32842->15999/tcp, 0.0.0.0:32844->3306/tcp, 0.0.0.0:15102->8080/tcp
-compose_vttablet103_1        sh -c /script/vttablet-up. ...   Up (healthy)   0.0.0.0:32841->15999/tcp, 0.0.0.0:32843->3306/tcp, 0.0.0.0:15103->8080/tcp
+(base) compose❯ docker-compose ps
+                Name                              Command                  State                                                  Ports
+----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
+compose_consul1_1                      docker-entrypoint.sh agent ...   Up             8300/tcp, 8301/tcp, 8301/udp, 8302/tcp, 8302/udp, 0.0.0.0:8400->8400/tcp,:::8400->8400/tcp,
+                                                                                       0.0.0.0:8500->8500/tcp,:::8500->8500/tcp, 0.0.0.0:8600->8600/tcp,:::8600->8600/tcp,
+                                                                                       8600/udp
+compose_consul2_1                      docker-entrypoint.sh agent ...   Up             8300/tcp, 8301/tcp, 8301/udp, 8302/tcp, 8302/udp, 8400/tcp, 8500/tcp, 8600/tcp, 8600/udp
+compose_consul3_1                      docker-entrypoint.sh agent ...   Up             8300/tcp, 8301/tcp, 8301/udp, 8302/tcp, 8302/udp, 8400/tcp, 8500/tcp, 8600/tcp, 8600/udp
+compose_external_db_host_1             docker-entrypoint.sh --ser ...   Up (healthy)   0.0.0.0:51016->3306/tcp, 33060/tcp
```

Comment:
[memo] I think external_db_host doesn't make sense in this compose configuration.
I might better delete it for the sake of simplicity.